Neo4j.Driver/Neo4j.Driver/Preview/GraphDatabasePreview.cs

```diff
@@ -0,0 +1,63 @@
+﻿// Copyright (c) "Neo4j"
+// Neo4j Sweden AB [http://neo4j.com]
+//
+// This file is part of Neo4j.
+//
+// Licensed under the Apache License, Version 2.0 (the "License").
+// You may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using Neo4j.Driver.Auth;
+
+namespace Neo4j.Driver.Preview;
+
+/// <summary>
+/// Contains preview methods for <see cref="GraphDatabase"/>. These methods are subject to change or removal in
+/// future versions.
+/// </summary>
+public class GraphDatabasePreview
```

Comment:
These say "connected to provided uri" but it doesn't accurately reflect the lazy connections of drivers,

Reply:
Updated